src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json

```diff
@@ -683,6 +757,17 @@
         "documentTypeCode": "BRL",
         "linkType": "kfs"
       }, {
+        "label": "CG Reporting Code",
+        "link": "kr/lookup.do?methodToCall=start&businessObjectClassName=edu.cornell.kfs.coa.businessobject.ContractGrantReportingCode&docFormKey=88888888&hideReturnLink=true",
+        "linkType": "kfs",
+        "permission": {
+          "templateNamespace": "KR-SYS",
+          "templateName": "Initiate Document",
```

Comment:
Well, I may have done this intentionally, but it's wrong, I'll fix it. Thanks for catching it.